Gemfile.lock

```diff
@@ -348,3 +346,6 @@ DEPENDENCIES
   watir-webdriver
   web-console (~> 2.0)
   whenever
+
+BUNDLED WITH
```

Comment:
can we remove these lines? bundler versions don't matter _that_ much and with this people won't be able to `bundle` unless they're using this exact version